src/main/java/seedu/address/model/person/insurance/InsurancePlan.java

```diff
@@ -12,6 +16,12 @@ public abstract class InsurancePlan {
      */
     protected int insurancePlanId = -1;
 
+    protected ArrayList<Claim> claims;
+
+    InsurancePlan() {
+        claims = new ArrayList<>();
```

Comment:
Should this be `ArrayList<Claim>` instead?

Reply:
I think not necessary lol but for greater clarity, it will be good.